src/hooks/useGiftTheme.ts

```diff
@@ -1,9 +1,16 @@
-import useApiRequest from './useApiRequest';
+import { useQuery } from '@tanstack/react-query';
 import { fetchThemes } from '@/api/ThemeApi';
 import type { GiftThemeType } from '@/types/theme';
 
 export default function useGiftTheme() {
-  const { data: themes, loading, error } = useApiRequest<GiftThemeType[], []>(fetchThemes, []);
+  const {
+    data: themes = [],
+    isLoading: loading,
+    isError: error,
+  } = useQuery<GiftThemeType[]>({
+    queryKey: ['themes'],
+    queryFn: fetchThemes,
+  });
 
   return { themes, loading, error };
```

Comment:
[제안] useQuery가 반환하는 객체 자체를 그대로 반환하는 방법도 고려해주세요~
```suggestion
  return useQuery(...);
```